package.json

```diff
@@ -62,12 +63,14 @@
     "@babel/polyfill": "^7.4.4",
     "classlist-polyfill": "^1.2.0",
     "curl": "cujojs/curl#0.8.13",
-    "d3": "^3.5.4",
+    "d3": "^6.2.0",
     "domready": "^1.0.7",
     "fastclick": "^1.0.6",
     "fence": "git://github.com/guardian/fence",
+    "index": "^0.4.0",
```

Comment:
Same as above, where is this used?

Reply:
again i don't think it is used, i can't find it being used.